socorro/external/rabbitmq/test_rabbitmq.py

```diff
@@ -1,6 +1,6 @@
 import pika
```

Comment:
I disagree, but it's possible that you know better. 
The `socorro/external/*/` directories feel pure and this is a script whose primary purpose in life is not for run-time stuff but only for testing.